packages/encryption/src/encryptionDevice.ts

```diff
@@ -566,8 +566,15 @@ export class EncryptionDevice {
     ): Promise<{ ciphertext: string; sessionId: string }> {
         return await this.cryptoStore.withGroupSessions(async () => {
             log(`encrypting msg with group session for stream id ${streamId}`)
-
-            checkPayloadLength(payloadString)
+            try {
+                checkPayloadLength(payloadString)
```

Comment:
can you add an `opts: { streamId?: string, source: string }` param to checkPayloadLength instead of this try/catch that does a dlogError before throwing so that we get logs from all the different places?

Reply:
good idea, thank you